config/config.go

```diff
@@ -62,7 +62,7 @@ func (config *Config) Validate(path string) ([]string, error) {
 		}
 		cameraName = config.Sensors[0]
 
-		if config.DataRateMsec < 0 {
+		if config.DataRateMsec != nil && *config.DataRateMsec < 0 {
```

Comment:
will the IMU run into a similar issue or is it fine to leave as is?

Reply:
This is a problem for lidars becauses  `data_rate_msec m = 0` needs to be differentiated from  `data_rate_msec m = nill` so that we can determine if we are in offline or online. I am not sure if there will be an alternate sensor process for imus or if we can rely on the lidarDataRateMsec to close both. @PStrutz Do you know?

Reply:
There will be a separate sensor process, but I can configure it to check the lidar data rate to determine offline/online mode

Reply:
I think that makes sense because we do not want to define the mode in multiple places @JohnN193 what do you think?

Reply:
yeah that makes sense, but for this pr replay cameras will still be broken for the imu integration feature flag. If we are okay with that & letting @PStrutz handling it when she gets to replay for imus then thats fine with me

Reply:
@JohnN193 after talking with @EshaMaharishi it sounds like `data_frequency_hz` when IMU ff is on is the new way to represent `data_rate_msec`. So I updated the checks to make it so if that is not provided we set it to the default value, if it is provided and not zero same behavior as before and if it is provided and is 0 we just set the `dataRateMsec` to 0. Let me know what you think now

cc @PStrutz